Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM node:18.19.1-slim
 
 # Define build arguments for repository branches
-ARG SHARDEUM_BRANCH=dev
-ARG RELAYER_COLLECTOR_BRANCH=dev
-ARG JSON_RPC_SERVER_BRANCH=dev
+ARG SHARDEUM_BRANCH=mainnet-launch
+ARG RELAYER_COLLECTOR_BRANCH=mainnet-launch
+ARG JSON_RPC_SERVER_BRANCH=mainnet-launch
+ARG NETWORK=custom
```

Comment:
any reason we can't have it as devnet or something?

Reply:
This would default the PR preview builds to `custom` which i think is what we would want.